conf/oscd.yaml

```diff
@@ -18,8 +18,8 @@ experiment:
     num_filters: 256
     ignore_index: 0
   datamodule:
-    root_dir: "data/oscd"
-    batch_size: 32
```

Comment:
The OSCD data module has never had a `batch_size` parameter. The correct name is `train_batch_size`. This parameter was being ignored since we never actually used `kwargs` before.